calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -95,19 +96,19 @@ def _configure_interface(self):
         return ep
 
     def _get_node_ip(self):
-        """Determine the IP for the host node.
-
-        This hits a well-known IP (the Google public DNS).
+        """ Determine the IP for the host node.
```

Comment:
Docstring here is a bit weirdly formatted now.  

Also no need to say "Does not use public internet" - that should be the default assumption :)